src/actions/users.test.js

```diff
@@ -0,0 +1,70 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { USER_INFO_REQUEST, USER_INFO_SUCCESS, USER_INFO_FAILURE, fetchUserInfo } from './users';
+import { ERROR } from './errors';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+const userInfoRequest = 'USER_INFO_REQUEST';
+const userInfoSuccess = 'USER_INFO_SUCCESS';
+const userInfoFailure = 'USER_INFO_FAILURE';
+
+const result = {
```

Comment:
You can remove this as it is duplicating the user object defined in `window.OPENSHIFT_CONFIG`. This means it would need to be updated in two places.